### PR TITLE
Fixes0229

### PR DESCRIPTION
**Pull request by prensing**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/AprilTagVision.java
Comment (Weepfrog) [inline src/main/java/frc/robot/subsystems/AprilTagVision.java:1]: need to add toRadians() on the yaw on the camera offsets

**Review by Weepfrog — COMMENTED** [review]
just one change. idk if someone is already on it

**Review by Weepfrog — APPROVED** [review]
Looks Good :D

**Comment by prensing** [comment]
Yes, you are right. Best to fix known bugs early. Done.

**Pull request merged**